recenter_tc/compare_outputs_recenter_tc.py

```diff
@@ -10,27 +10,30 @@
 # # # for more details. If you did not receive the license, for more information see:
 # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-""" Test script for running representative products using data and comparison outputs from geoips_test_data\_\* """
+"""Test script for running representative products.
 
-import subprocess
+Uses data and comparison outputs from geoips_test_data/_/*
+"""
 import logging
-from os.path import splitext
 
 from geoips.compare_outputs import compare_outputs, test_product, text_match
 
 LOG = logging.getLogger(__name__)
 
 
 def is_fdeck(fname):
-    """Check if fname is an fdeck file
+    r"""Check if fname is an fdeck file.
```

Comment:
I don't think it really makes a difference, but why did this need to become a raw string?

Reply:
I took out the "r", @AJThorpe was there any reason the "r" got added?

Reply:
./recenter_tc/__init__.py:58:1: F401 '._version.__version__' imported but unused
  ./recenter_tc/__init__.py:58:1: F401 '._version.__version_tuple__' imported but unused
  ./recenter_tc/compare_outputs_recenter_tc.py:[25](https://github.com/NRLMMD-GEOIPS/recenter_tc/actions/runs/6537613353/job/17751817532?pr=28#step:10:26):1: D[30](https://github.com/NRLMMD-GEOIPS/recenter_tc/actions/runs/6537613353/job/17751817532?pr=28#step:10:31)1 Use r""" if any backslashes in a docstring
  ./recenter_tc/compare_outputs_recenter_tc.py:35:38: W605 invalid escape sequence '\*'
  1     D301 Use r""" if any backslashes in a docstring
  2     F401 '._version.__version__' imported but unused
  1     W605 invalid escape sequence '\*'

Reply:
This works now (``*_FIX`` instead of \", which removed the need for "r")